components/molecules/HighlightInput/highlight-input-form.tsx

```diff
@@ -527,7 +527,7 @@ const HighlightInputForm = ({ refreshCallback }: HighlightInputFormProps): JSX.E
         }}
         open={isDivFocused}
       >
-        <DialogContent className="p-4 w-full md:w-[33rem] xs:w-[25rem] max-h-[80vh]">
+        <DialogContent className="p-4 w-full md:w-[40rem] xs:w-[25rem] max-h-[80vh] overflow-y-scroll">
```

Comment:
I don't think we need `md:w-[40rem] xs:w-[25rem]`. I removed them and it still looked fine. We could add `m-4` except for a small screen and that should be fine.

Reply:
Taking those off makes the content grow to the screen because we have a default of w-full set in the root here https://github.com/open-sauced/app/blob/2b7b61391d73081ea1fcb403ccb1883b636220ad/components/molecules/Dialog/dialog.tsx#L56

![image](https://github.com/open-sauced/app/assets/62995161/3f892b82-1e67-4c92-ae4e-8a04f7fef8a8)

Reply:
Maybe use `w-max` instead of `w-full`? If not, we'll keep as is. Just trying to avoid the Tailwind anti-pattern of explicit values.